Source/WebCore/platform/graphics/ca/cocoa/PlatformCALayerCocoa.h

```diff
@@ -231,7 +231,7 @@ class PlatformCALayerCocoa final : public PlatformCALayer {
     ScrollingNodeID m_scrollingNodeID;
 #endif
     EventRegion m_eventRegion;
-    bool m_wantsDeepColorBackingStore { false };
+    ContentsFormat m_contentsFormat { ContentsFormat::RGBA8 };
```

Comment:
You may need to #include ContentsFormat.h.

Reply:
Fixed.